api_tests/users/views/test_user_actions.py

```diff
@@ -1,4 +1,6 @@
 import pytest
+import mock
+from nose.tools import assert_false, assert_true
```

Comment:
Honest question: Is there a reason to use `assert_true`/`assert_false` over plain `assert`/`assert not`?